.github/workflows/on_bundle_update_available.yaml

```diff
@@ -0,0 +1,36 @@
+name: bundle update available
+
+on:
+  schedule:
+    - cron:  '0 */8 * * 1-5'
+
+jobs:
+  update-bundles:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: check for updates
+        run: |
+          ./update_bundle.py releases/latest/mysql-k8s-bundle.yaml
+      - name: create pull request
+        id: cpr
+        uses: peter-evans/create-pull-request@v4
+        with:
+          token: ${{ secrets.PAT }}
+          commit-message: Update bundles
+          committer: GitHub <noreply@github.com>
+          author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+          signoff: false
+          branch: update-bundles
+          delete-branch: true
+          title: 'Update Bundles'
+          body: |
+            Update bundles with new revisions from Charmhub.
+          labels: |
+            automated pr
+          assignees: team-reviewers, shayancanonical, paulomach
+          reviewers: team-reviewers, shayancanonical, paulomach
```

Comment:
Remember you're going to need to add a charmhub token and a github PAT to this repo. I don't think you can self-review PRs, so if you're using a PAT from your own account, you need to remove yourself as a reviewer. Don't worry, you'll still get daily notifications ;)

Reply:
Great call! Created and added the tokens to the repository secrets